src/Microsoft.AspNetCore.Session/DistributedSession.cs

```diff
@@ -65,6 +71,33 @@ public class DistributedSession : ISession
             _isNewSessionKey = isNewSessionKey;
         }
 
+        public string Id
+        {
+            get
+            {
+                Load(); // TODO: Silent failure
```

Comment:
Load isn't really needed here since it doesn't set the _sessionId and IdBytes will call Load if needed